stackdriver.go

```diff
@@ -0,0 +1,54 @@
+package logformatter
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/golang/protobuf/jsonpb"
+	"google.golang.org/genproto/googleapis/devtools/clouderrorreporting/v1beta1"
+	"google.golang.org/genproto/googleapis/logging/v2"
+)
+
+type Stackdriver struct {
+	logging.LogEntry
+	clouderrorreporting.ErrorEvent
+	Payload map[string]interface{}
+}
+
+func (s *Stackdriver) MarshalJSON() ([]byte, error) {
+	var (
+		m jsonpb.Marshaler
+	)
+
+	unwrap := func(m []byte) []byte {
```

Comment:
Could we have some comments here?
Maybe provide some use cases to explain what it aims to do?

Reply:
I could add the comments here which might be clear.

Reply:
done